composer.json

```diff
@@ -41,17 +41,13 @@
             },
```

Comment:
Ok, back on here. 

Can I double check that we agree on creating a branch named say drupal-8.4.x so that I can make reference to it in the docs?